docs/conf.py

```diff
@@ -20,7 +20,7 @@
 # directory, add these directories to sys.path here. If the directory is
```

Comment:
In this file, I would instead all the sections with commented-out code.

They exist to make the initial configuration file creation easy, but they are not needed after that, and they can even be misleading (as you could think these are all the settings there are, but there may be upstream changes that add, remove or change the default values of some of these settings).

Reply:
agree!